CHANGELOG.md

```diff
@@ -21,8 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Improve cleanup of `DrainerConfig` CRs after node draining.
 - Use release.Revision in Helm chart for Helm 3 support.
-
-
+- Disable profiling for Controller Manager and Scheduler.
```

Comment:
This here does not look right.

Reply:
It looks like this was actually changed in [v6.0.3](https://github.com/giantswarm/k8scloudconfig/blob/master/CHANGELOG.md#v603-2020-04-16) so I will move it to unreleased. Thanks.